src/frameworks/drupal8/solr.md

```diff
@@ -1,71 +1,62 @@
-
-First you need to create a Solr service.  In your `.platform/services.yaml` file, add or uncomment the following:
+Add the following to your `.platform/services.yaml` file.
 
 ```yaml
-solrsearch:
-    type: solr:6.6
+search:
+    type: solr:8.0
     disk: 1024
     configuration:
         cores:
             maincore:
-                conf_dir: !archive "solr-conf/6.x"
+                conf_dir: {}
         endpoints:
             main:
                 core: maincore
 ```
 
-The above definition defines a single Solr 6.6 server.  That server has 1 core defined: `maincore` - the configuration for which is in the `.platform/solr-conf/6.x` directory.
+The above definition defines a single Solr 8.0 server.  That server has 1 core defined: `maincore`, which will use a default configuration.  (The default configuration is not suitable for production but will allow the module to connect to it.)
 
 It then defines one endpoint: `main` is connected to the `maincore`.
 
-### Expose the Solr service to your application
+### 3. Expose the Solr service to your application
 
-In your `.platform.app.yaml` file, we now need to open a connection to the new Solr service.  Under the `relationships` section, add or uncomment the following:
+In your `.platform.app.yaml` file, we now need to open a connection to the new Solr service.  Under the `relationships` section, add the following:
 
 ```
 relationships:
-    solr: 'solrsearch:main'
+    solrsearch: 'search:main'
 ```
 
-That is, the application's environment would include a `solr` relationship that connects to the `main` endpoint, which is the `maincore` core.
+That is, the application's environment would include a `solrsearch` relationship that connects to the `main` endpoint, which is the `maincore` core.
 
-The key (left side) is the name that will be exposed to the application in the `PLATFORM_RELATIONSHIPS` [variable](/development/variables.md).  The right hand side is the name of the service we specified above (`solrsearch`) and the endpoint (`main`).  If you named the service something different above, change `solrsearch` to that. The same rule is valid for the value of the endpoint named `main`.
+The key (left side) is the name that will be exposed to the application in the `PLATFORM_RELATIONSHIPS` [variable](/development/variables.md).  The right hand side is the name of the service specified above (`search`) and the endpoint (`main`).  If you named the service or endpoint something different above, change those values accordingly.
 
-## Configuration
+### 4. Add auto-configuration code to `settings.platformsh.php`
 
-The configuration can be managed from `settings.platformsh.php` by adding the following code snippet.
+The configuration can be managed from `settings.platformsh.php` by adding the following code snippet.  It will override the environment-specific parts of the configuration object with the correct values to connect to the Platform.sh Solr instance.
 
 > **note**
 >
```

Comment:
Can't make a suggestion here from this view of the diff, but maybe it would be a good idea to include a link here https://github.com/platformsh/config-reader-php for someone who's gotten this far but doesn't know about these yet

Reply:
I think the composer install command is sufficient.  This is largely copy-pasta instructions, so it's just enough to support the pasta.